plugins/rrl/handler.go

```diff
@@ -26,11 +27,14 @@ func (rrl *RRL) ServeDNS(ctx context.Context, w dns.ResponseWriter, r *dns.Msg)
 
 	// Limit request rate
 	if rrl.requestsInterval != 0 {
+		// We multuply because allowance is (1s / RPS)
+		requestsInterval := rrl.requestsInterval * atomic.LoadInt64(&rrl.podCount)
```

Comment:
Do we need some minimum to make sure the request limit doesn't get too low?

Reply:
Ah yeah we should min pod count at 1

Reply:
That sounds good. But I said the wrong thing in my above comment. What I meant was, what if the requestsInterval gets so high that the individual pods effectively serve no traffic?

Reply:
We do have a maximum number of pods we allow the deployment to scale to, we should be able to account for that when selecting our new ratelimit values.